test/library/draft/Vector/PREDIFF

```diff
@@ -0,0 +1,16 @@
+#!/bin/python
```

Comment:
`/usr/bin/env python`

What you have ties us to the system's default python, but we want to be able to use the one with setup for the testing.

You can run `find test -name "PREDIFF" | xargs -L1 grep python` in Chapel root to see for yourself.